src/codec/framed_write.rs

```diff
@@ -105,8 +105,10 @@ where
     pub fn buffer(&mut self, item: Frame<B>) -> Result<(), UserError> {
         // Ensure that we have enough capacity to accept the write.
         assert!(self.has_capacity());
+        let span = tracing::trace_span!("FramedWrite::buffer", frame = ?item);
```

Comment:
I noticed some of the spans are prefixed with the struct name, and others aren't. Is one preferred over the other? Should we be consistent?

Reply:
I've seen span names that are prefixed, and span names that are not prefixed. I think it depends somewhat on whether the name of the function is unambiguous on its own, or if the name of the type is necessary to make the meaning clear. e.g. if there is only one `handshake` function, it doesn't need a prefix, but a `poll` function definitely needs a prefix, because there are lots of `poll`s. 

I've tried to err on the side of brevity and used un-prefixed names when they don't seem strictly necessary, and only use prefixed names when they do. Brevity seemed worthwhile, because at the highest verbosity levels, the span contexts can get quite long. However, if folks prefer, I'm happy to err on the side of consistency rather than brevity.